### PR TITLE
[Doc] Use pipeline to run test

### PR DESCRIPTION
**Pull request by qwordy**
**Description of PR (Mandatory)**  
(Why this PR? What is changed? What is the effect? etc. A high-quality description can accelerate the review process)
Add an article about how to use Azure pipeline to run test.

**Testing Guide**  
(Example commands with explanations)

**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S168

File: doc/use_azure_pipeline_to_run_test.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Use Azure Pipelines to Run Tests
```
Comment (yungezz) [inline doc/use_azure_pipeline_to_run_test.md:1]: thinking should this doc be public in github, as it's used for our daily development and the pipeline is inside our org.

File: doc/use_azure_pipeline_to_run_test.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+This tutorial shows how to use Azure Pipelines to test Azure CLI commands. The usage may be changed in the future.
```
Comment (yungezz) [inline doc/use_azure_pipeline_to_run_test.md:3]: worth to highlight the pipeline is used for live test

Comment (qwordy) [reply doc/use_azure_pipeline_to_run_test.md:3]: OK

File: doc/use_azure_pipeline_to_run_test.md (hunk 1 above)
Comment (qwordy) [reply doc/use_azure_pipeline_to_run_test.md:1]: Yes, it should be.

**Comment by qwordy** [comment]
Close it. Move it to internal platform.

**Pull request closed without merging**